listenbrainz/webserver/views/api.py

```diff
@@ -119,7 +119,7 @@ def get_listens(user_name):
     latest_listen = db_conn.fetch_listens(
         user_name,
         limit=1,
-        to_ts=max_ts,
+        to_ts=int(time.time()),
```

Comment:
I think it'd make sense to only calculate time.time() once (it's also used if max_ts and min_ts aren't set)